ibm_catalog.json

```diff
@@ -170,7 +170,27 @@
                   ]
                 },
                 {
-                  "key": "elasticsearch_version"
+                  "key": "elasticsearch_version",
+                  "required": true,
+                  "default_value": "",
```

Comment:
don't add any `default_value` - let it default the the terraform default value

Reply:
done.